src/cascadia/TerminalApp/App.cpp

```diff
@@ -1052,10 +1053,17 @@ namespace winrt::TerminalApp::implementation
     // Arguments:
     // - trimTrailingWhitespace: enable removing any whitespace from copied selection
     //    and get text to appear on separate lines.
-    void App::_CopyText(const bool trimTrailingWhitespace)
+    // Return Value:
+    // - true iff we we able to copy text (if a selection was active)
+    bool App::_CopyText(const bool trimTrailingWhitespace)
     {
         const auto control = _GetFocusedControl();
-        control.CopySelectionToClipboard(trimTrailingWhitespace);
+        if (control.HasSelection())

```

Comment:
This just feels weird. Like, SHOULD the app be aware of whether the control has a selection or not? I feel like that responsibility falls on the control, and the control should tell the app what happened.

Won't this become a problem if the `TermControl` is split from the `TermApp`?